commons-mfa/src/main/java/org/exoplatform/mfa/rest/mfa/MfaRestService.java

```diff
@@ -21,10 +23,15 @@
 @Api(value = "/mfa")
 public class MfaRestService implements ResourceContainer {
 
+  public static final String MFA_FEATURE = "mfa";
+
   private MfaService mfaService;
 
-  public MfaRestService(MfaService mfaService) {
+  private ExoFeatureService featureService;
```

Comment:
No need to have feature service here, as you can have mfaService.isFeatureActive